shared/authplugin/src/main/java/io/pravega/auth/AuthHandler.java

```diff
@@ -14,7 +14,7 @@
 /**
  * Custom authorization/authentication handlers implement this interface.
  * The implementations are loaded from the classpath using `ServiceLoader` (https://docs.oracle.com/javase/7/docs/api/java/util/ServiceLoader.html)
- * Pravega controller also implements this interface through {@link io.pravega.controller.server.rpc.auth.PasswordAuthHandler}.
+ * Pravega controller also implements this interface through {@code io.pravega.controller.server.rpc.auth.PasswordAuthHandler}.
```

Comment:
PasswordAuthHandler is part of the `controller` module. `shared/authplugin` doesn't have a dependency on `controller` module, so javadoc can't refer to PasswordAuthHandler. So I have used `@code` tag here instead of `@link`.

Reply:
Sounds good.